code/src/terrat_vcs_service_github/terrat_vcs_service_github_provider.ml

```diff
@@ -2012,6 +2015,7 @@ module Apply_requirements = struct
                   {
                     Result.passed;
                     match_;
+                    apply_after_merge = (if apply_after_merge.Afm.enabled then Some merged else None);
```

Comment:
@orbitz I need more context here, for now I decided to propagate an option bool and (as of now) it only propagates whether the PR is merged or not.